apps/explorer/src/components/validator-map/index.tsx

```diff
@@ -85,23 +76,26 @@ export default function ValidatorMap({ minHeight }: Props) {
         return count;
     }, [nodeData]);
 
-    useEffect(() => {
-        setIsLoading(true);
-        appsBe(`validator-map`, {
-            network: network.toLowerCase(),
+    const {
+        data: validatorData,
+        isLoading,
+        isError,
+    } = useQuery(['validator-map'], async () =>
+        appsBe(
+            `validator-map`,
+            {
+                network: network.toLowerCase(),
+            },
+            {
+                cache: 'no-cache',
```

Comment:
Why no-cache?

Reply:
left that in by mistake for testing, will remove